lib/whitehall_importer/create_edition.rb

```diff
@@ -184,18 +186,62 @@ def create_revision_history(edition)
         entry_type = history.imported_entry_type(event, edition_number)
         next if entry_type.nil?
 
-        details = TimelineEntry::WhitehallImportedEntry.create!(
-          entry_type: entry_type,
-        )
-        TimelineEntry.create!(
-          entry_type: :whitehall_migration,
-          created_at: event["created_at"],
-          created_by_id: user_ids[event["whodunnit"]],
-          edition: edition,
-          document: edition.document,
-          details: details,
-        )
+        details = create_timeline_import_entry(entry_type)
+        create_timeline_entry(details, edition, event["created_at"], user_ids[event["whodunnit"]])
       end
     end
+
+    def create_notes(edition)
+      whitehall_edition["editorial_remarks"].each do |event|
+        contents = {
+          contents: event["body"],
```

Comment:
shouldn't this be a body key here?

Reply:
As in `body: event["body"]`?  I think I'm getting a tad confused, but the only place we really need `body` is in the `event`, so I've just updated everything else (to be consistent) to `contents`.